src/app/actions.js

```diff
@@ -444,5 +446,151 @@ export let actions = () => ({
+			},
+			{
+				time: 12,
+				id: 13,
+				cname: 2,
+				author: {
+					id: 123,
+					name: "Afroz kana"
+				},
+				text: "TI feel like "
+			},
+			{
+				time: 13,
+				id: 12,
+				cname: 2,
+				author: {
+					id: 123,
+					name: "Afroz kana"
+				},
+				text: "TI feel like we saw this shot twice. Maybe we could try cutting to a different angle"
+			},
+			{
+				time: 25,
+				id: 3,
+				cname: 2,
+				author: {
+					id: 12,
+					name: "Afroz"
+				},
+				text: "TI feel like we saw this shot twice. Maybe we could try cutting to a different angle kw knowledge 2"
+			},
+			{
+				time: 35,
+				id: 4,
+				cname: 2,
+				author: {
+					id: 123,
+					name: "Afroz kaana"
+				},
+				text: "test keywords kw knowledge 1"
+			},
+			{
+				time: 40,
+				id: 41111,
+				cname: 2,
+				author: {
+					id: 12,
+					name: "Afroz"
+				},
+				text: "TI feel like we saw this shot twice. Maybe we could try cutting to a different angle"
+			},
+			{
+				time: 60,
+				id: 12564,
+				cname: 2,
+				author: {
+					id: 123,
+					name: "Afroz kaana"
+				},
+				text: "TI feel like we saw this shot twice. Maybe we could try cutting to a different angle kw clarity 1"
+			}
+		];
+
+		return {
+			...state,
+			transcriptionPane: {
+				...state.transcriptionPane,
+				timestampedTranscripts: commentArray,
+				searchedTranscripts: commentArray
+			}
+		}
+	},
+
+	updateTranscriptionSearchWords: (state, { searchWords }) => {
+		let searchKeywords = transcriptionUtils.getKeywordsInParams(state.transcriptionPane.filter.selectedEvalParams);
+		let allWords = searchWords.concat(searchKeywords);
+
+		let { searchedTranscripts, matchedTranscriptIndices } = 
+			transcriptionUtils.search(state.transcriptionPane.timestampedTranscripts, allWords);
+
+		return {
+			...state,
+			transcriptionPane: {
+				...state.transcriptionPane,
+				searchBar: {
+					...state.transcriptionPane.searchBar,
+					searchWords: searchWords,
+					currentMatchNumber: (matchedTranscriptIndices.length == 0) ? 0 : 1,
+					numberOfMatches: matchedTranscriptIndices.length
+				},
+				searchedTranscripts: searchedTranscripts,
+				matchedTranscriptIndices: matchedTranscriptIndices
+			}
+		}
+	},
+
+	navigateToMatchNum:(state, { currentMatchNumber }) => {
+		return {
+			...state,
+			transcriptionPane: {
+				...state.transcriptionPane,
+				searchBar: {
+					...state.transcriptionPane.searchBar,
+					currentMatchNumber: currentMatchNumber,
+				}
+			}
+		}
+	},
+
+	updateTranscriptionFilters: (state, { selectedEvalParams }) => {
```

Comment:
merge this with updateTranscriptionSearchWords method

Reply:
Done